src/modules/render.js

```diff
@@ -1,21 +1,97 @@
-import tasks from './array.js';
-
-const renderTasks = () => {
-  const toDoList = document.getElementById('to-do-list');
-  toDoList.innerHTML = '';
-  tasks.sort((a, b) => a.index - b.index);
-  toDoList.innerHTML = tasks
-    .map(
-      (task) => `
-    <li class='to-do-item'>
-      <button class='check-button'>
-      </button>
-      <span class='task-description'>${task.description}</span>
-      <span class='fas fa-ellipsis-v'></span>
-    </li>
-  `,
-    )
-    .join('');
-};
-
-export default renderTasks;
+import { form, input } from './link.js';
+
+export const menuIcons = document.querySelectorAll('.menu-icon');
+export class ToDoList {
+  constructor() {
+    this.collection = JSON.parse(localStorage.getItem('bookCollection')) || [];
+    if (!Array.isArray(this.collection) || this.collection.length === 0) {
+      this.collection = [];
+    }
+    this.renderTasks();
+    form.addEventListener('submit', (event) => {
+      event.preventDefault();
+      this.addBook(input.value);
+      input.value = '';
+    });
+  }
+
+  addBook = (Description) => {
+    const newTask = {
+      description: Description,
+      completed: false,
+      index: this.updateIndex(),
+    };
+    this.collection.push(newTask);
+    localStorage.setItem('bookCollection', JSON.stringify(this.collection));
+    this.renderTasks();
+  };
+
+  removeBook = (index) => {
+    this.collection[index].markedForDeletion = true;
+    this.collection = this.collection.filter((task, i) => i !== index);
+    localStorage.setItem('bookCollection', JSON.stringify(this.collection));
+    this.renderTasks();
+  };
```

Comment:
- [ ] Please 🙏🏽 update the indexes after removing the tasks to represent the correct content of the new array. 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/229007172-2f18b05e-e7be-47f6-a8ea-7c90fb26b342.png)

- **[OPTIONAL]** We could use a meaningful name for the functions e.g. `task` instead of `book`. 😉